docs/CONTRIBUTOR_GUIDE.md

```diff
@@ -34,6 +34,19 @@ and run:
 goimports -local sigs.k8s.io/krew -w cmd pkg integration_test
 ```
 
+Shell scripts are automatically formatted by `shfmt`, to install and to validate run:
+
+```bash
+hach/run-lint.sh
+```
+
+If there is a divergence, diff will be provided. Otherwise, no output.
+Diff can be fixed by running:
+
+```bash
+shfmt -w -i=2 **/*.sh
```

Comment:
I have a suspicion that `**` does nothing on bash 3.x (requires `shopt -s globstar`), it's fine on bash 4.x. Also, would be good to limit this to only `hack/`?

Reply:
Yeah let's do this `hack/` because the validating script (`run-lint.sh`) is also just running on hack/.